WordPress/Classes/ViewRelated/Reader/ReaderPostsViewController.m

```diff
@@ -870,8 +840,13 @@ - (void)cleanupAfterRefresh
 
 - (void)tableViewHandlerWillRefreshTableViewPreservingOffset:(WPTableViewHandler *)tableViewHandler
 {
-    [UIView performWithoutAnimation:^{
-        [self cleanupAfterRefresh];
+    [[self managedObjectContext] performBlockAndWait:^{
```

Comment:
Hmm... Now that I look at it again I'm not sure why we'd need a perform block at all here.  Thanks for flagging it!